### DISCUSSION TITLE
Add Fla’s to toonsquid

### DISCUSSION THREAD

**Discussion by Storm840** [keiwando/toonsquid#554] [Ideas]
With toonsquid getting better and better, i think it would be really good to be able to import fla's into toonsquid.
(Fla's are Adobe animate files for those who don't know)

**Comment by VeiloPot** [comment]
This is a really interesting idea, but at the same time i feel like it would be too complex to be imported in Toonsquid. Because even if both programs are a little similar in one way, i still don't see how this might happen.

**Comment by b1zarr3vel** [comment]
i think it would also be good if toonsquid could export projects as fla files to import into animate

**Reply by ArtsyStar1** [reply]
Yeah, that would be nice! I hope the next update will include these features.

**Comment by GEmoon181** [comment]
It would be good so I can now get my bfdi :)

**Comment by Lioncub901** [comment]
I need this for college

**Comment by DWrightAnimation** [comment]
I personally don't use Adobe Animate myself, but this feature would help ToonSquid be incorporated into the industry.
